### ISSUE TITLE
BatchTask produces nonsensical log messages

### ISSUE THREAD

**Issue by logological** [dkpro/dkpro-lab#77]
The DKPro Lab `BatchTask` produces logging messages such as the following:

````
2015-07-17 15:19:31  INFO [main] (BatchTask) - Completed configuration 2 of 1 (200%  ETA 00:00:-9.-165  RUN 00:00:18.331  AVG 9166  LAST 8274)
````

There are a couple problems here — first, in the "Completed configuration _x_ of _y_" part, we have _x_ > _y_.  Second, the ETA has negative values for the seconds and milliseconds.

I was able to reproduce this problem using several different experiments.  For a simple and ready-made one, run `de.tudarmstadt.ukp.dkpro.tc.groovyexamples.single.unit.BrownUnitPosDemo` from DKPro TC.  (Be sure to adjust the logging level first so that DKPro Lab's messages are visible.)

**Comment by daxenberger** [comment]
This problem occurs in DKPro TC Cross-Validation setups (as a product of overrding the execute() method of the BatchTask). Maybe the FoldDimensionBundle which we use to split the input data into n folds/dimensions isn't properly handeled by the ProgressMeter?

**Comment by reckart** [comment]
Sounds like a bug in TC rather than in Lab. The ProgressMeter just shows what it's being told. If it is getting told that there are 100 things to do but 200 things that have been done, then the output will be as you see it. I don't think that ProgressMeter should start throwing exceptions in such a case. What would you expect?

**Comment by daxenberger** [comment]
No, ProgressMeter itself is not the problem. Either, we are using FoldDimensionBundle (used for the split into folds in crossvalidation setups) in DKPro TC incorrectly, or the execute() Method in BatchTask, which calculates the expected size for the ProgressMeter (ll. 154ff in BatchTask), does not handle Dimensions from FoldDimensionBundles correctly.
